modules/apps/calendar/calendar-web-test/src/testFunctional/tests/EventDetails.testcase

```diff
@@ -282,8 +282,8 @@ definition {
 			startDate = "02/2/2016");
 
 		CalendarEvent.setTimes(
-			endDateTime = " 2:00 AM",
-			startDateTime = " 1:00 AM");
+			endDateTime = "02:00 AM",
```

Comment:
The space that had been added due to the previous change was removed and the "0" was added in its place.